docs/quick_start.rst

```diff
@@ -230,27 +236,124 @@ Connect the new application to the OMF Connector Relay by selecting the new Fled
+   - **Server hostname:** The hostname or address of the PI Server.
+   - **Server port:** The port the PI Web API OMF endpoint is listening on. Leave as 0 if you are using the default port.
+   - **Data Source:** Defines which data is sent to the PI Server. The readings or Fledge's internal statistics.
+   - **Static Data:** Data to include in every reading sent to PI.  For example, you can use this to specify the location of the devices being monitored by the Fledge server.
+- Asset Framework
+   - **Asset Framework Hierarchies Tree:** The location in the asset framework into which the data will be inserted. All data will be inserted at this point in the asset framework unless a later rule overrides this.
+   - **Asset Framework Hierarchies Rules:** A set of rules that allow specific readings to be placed elewhere in the asset freamework. These rules can be based on the name of the asset itself or some metadata associated with the asset.
+- PI Web API authentication
+   - **PI Web API Authentication Method:** The authentication method to be used, anonymous equates to no authentication, basic authenetication requires a user name and password and Kerberos allows integration with your single sign on environment.
+   - **PI Web API User Id:**  The user name to authenticate with the PI Web API.
+   - **PI Web API Password:** The password of the user we are using to authenticate.
+   - **PI Web API Kerberos keytab file:** The Kerberos keytab file used to authenticate.
+- Connection management (These should only be changed with guidance from support)
+   - **Sleep Time Retry:** Number of seconds to wait before retrying the HTTP connection (Fledge doubles this time after each failed attempt).
+   - **Maximum Retry:** Maximum number of times to retry connecting to the PI server.
+   - **HTTP Timeout:** Number of seconds to wait before Fledge will time out an HTTP connection attempt.
+- Other (Rarely changed)
+   - **Integer Format:** Used to match Fledge data types to the data type configured in PI. This defaults to int64 but may be set to any OMF data type compatible with integer data, e.g. int32.
+   - **Number Format:** Used to match Fledge data types to the data type configured in PI. The defaults is float64 but may be set to any OMF datatype that supports floating point values.
+   - **Compression:** Compress the readings data before sending it to the PI System.
+
+EDS OMF Endpoint
+----------------
+
+To use the OSISoft Edge Data Store first install Edge Data Store on the same machine as your Fledge instance. It is a limitation of Edge Data Store that it must reside on the same host as any system that connects to it with OMF.
+
+Now go to the Fledge user interface, create a new North instance and select the “OMF” plugin on the first screen.
 The second screen will request the following information:
 
-+--------------------+
-| |pi_plugin_config| |
-+--------------------+
++---------------------+
+| |omf_plugin_config| |
```

Comment:
This image has with endpoint `OSI Cloud Services` selected. It should be with `Edge Data Store`